### DISCUSSION TITLE
Table Column Fixed issue when scrolling

### DISCUSSION THREAD

**Discussion by kenfoongAK** [rsuite/rsuite#3122] [Q&A]
Hi team,

When i tried to use the table column fixed and scroll to right, it will have blank column appear on the left. May refer the screenshot below :
<img width="1107" alt="Screenshot 2023-03-17 at 7 03 17 PM" src="https://user-images.githubusercontent.com/42329805/225888447-c25842f8-b3be-406b-950f-4f7c827025ab.png">

**Comment by simonguo** [comment]
Hi @kenfoongAK 
What rsuite version are you using?

**Reply by kenfoongAK** [reply]
is ^5.26.1

**Reply by simonguo** [reply]
A scroll bar position bug has been fixed in rsuite@5.28.3 https://github.com/rsuite/rsuite/pull/3116, you can try to upgrade rsuite to the latest version 5.28.3 to check if the problem still exists. If it doesn't solve the problem, please reproduce the problem through codesandbox.

**Reply by kenfoongAK** [reply]
the bug still come when I update version to rsuite@5.28.3 
I have reproduce the problem on https://codesandbox.io/p/sandbox/weathered-star-t2bn1l?file=%2FREADME.md
 May refer the screenshot below :

<img width="695" alt="image" src="https://user-images.githubusercontent.com/42329805/226236830-357fe5bd-ee54-4f68-81d5-99ac34d99e29.png">

**Reply by simonguo** [reply]
https://user-images.githubusercontent.com/1203827/226273463-202ca319-6319-46b2-8a7a-acc8f109927c.mp4

I can't reproduce your problem, please provide more information, such as: operating system, browser version, and how you operate.

**Reply by kenfoongAK** [reply]
OS: MacBook Pro 13.1
Browser: Google Chrome 111.0.5563.64
Below video is how I operate the bug

https://user-images.githubusercontent.com/42329805/226286398-13d4009e-18fc-4ab6-9940-ff8efcabb0e8.mov

**Reply by kenfoongAK** [reply]
Hi may i ask any update?

**Reply by simonguo** [reply]
It does only happen on browsers on some operating systems. You can try disabling `translate3d` to see if that fixes it.
```tsx
<Table translate3d={false} />
```

**Reply by kenfoongAK** [reply]
tq is work for me now, may I ask what is the function of translate3d

**Reply by simonguo** [reply]
`translate3d` can use GPU for rendering to achieve CSS3 hardware acceleration. But there may be problems with browser compatibility.
